src/main/java/org/tikv/common/ConfigUtils.java

```diff
@@ -93,6 +93,12 @@
   public static final String TIKV_KEY_CERT_CHAIN = "tikv.key_cert_chain";
   public static final String TIKV_KEY_FILE = "tikv.key_file";
 
+  public static final String TIKV_USE_JKS = "tikv.use_jks";
```

Comment:
how about `tikv.jks_enable`?

Reply:
hmm for using it in command line param in TIDB MR connector, I thought use JKS was more intuitive with TLS enable.

Reply:
is that ok?

Reply:
sure. `tikv.use_jks` SGTM